packages/core/src/common/i18n/messages.ts

```diff
@@ -0,0 +1,3 @@
+require('./i18n/en.json');
+require('./i18n/es.json');
+require('./i18n/fr.json');
```

Comment:
Yes, it is a left over, file not needed anymore, so I remove it from each package where it was defined